sw/device/freestanding/meson.build

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This target exists so that Meson can be made aware that other targets depend
+# on these eight headers.
```

Comment:
Again, maybe best to remove the word "eight".

Reply:
Done.